index.php 4

KachiAlex opened the pull request:
Very interesting. My experience with Favcode is ever increasingly enlightening and challenging. An honor to be here.

KachiAlex committed:
Update Felix - FavCode Test.html

favourori commented:
Seen. Visit the Read me again and Deploy your site to Heroku

KachiAlex committed:
Rename Felix - FavCode Test.html to index.php 4

Achemsamuel commented:
Okay, I created the heroku app and i have deployed...
https://sammy-favcode.herokuapp.com/

KachiAlex committed:
Rename index.php 4 to index.php4

Am effecting these changes because heroku is rejecting the GitHub account that it couldn't recognize the buildpack log

KachiAlex committed:
Update index.php4

Achemsamuel commented:
Try to change your edit to just index.php ..... leave the '4' out ...

KachiAlex committed:
Rename index.php4 to index.php

KachiAlex committed:
Update index.php

KachiAlex committed:
Update index.php
